definitions/npm/lodash_v4.x.x/flow_v0.104.x-/test_lodash-v4.x.x.js

```diff
@@ -202,56 +231,128 @@ describe('Collection', () => {
   it('countBy', () => {
     (countBy([6.1, 4.2, 6.3], Math.floor): { [string]: number, ... });
     (countBy(["one", "two", "three"], "length"): { [string]: number, ... });
+    (countBy(indexerObject, (v: number) => v): { [string]: number, ... });
+    (countBy(exactObject, (v: number) => v): { [string]: number, ... });
+    (countBy(exactHeterogeneousObject, (v: number | string | boolean) => v): { [string]: number, ... });
+    (countBy(readOnlyExactObject, (v: number) => v): { [string]: number, ... });
+
     // $ExpectError
     (countBy(["one", "two", "three"], "length"): { [string]: string, ... });
+    // $ExpectError
+    countBy(123, "length");
+    // $ExpectError
+    countBy(["one", "two", "three"], (v: boolean) => v);
+    // $ExpectError
+    countBy(["one", "two", "three"], (v, arg2: number) => arg2);
   });
 
+  /**
+   * Using such functions allows to:
+   * 1) Ensure all tests are run and up to date for each function
+   * 2) Enforce that type definitions for all related functions (e.g. each, eachRight, forEach, forEachRight) are the same, as they should be by the docs.
+   */
```

Comment:
I've wanted to addd this as a PR comment initially, but then thought that the purpose will be clear for more people if it's put in code